### PR TITLE
Fix the default value of BUILD_DATE.

### PR DESCRIPTION
**Pull request by googollee**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -29,7 +29,7 @@ COPY ui/package.json ui/package-lock.json /app/ui
 RUN npm ci
 
 COPY ui/ /app/ui
-RUN if [ "$BUILD_DATE" = "" ]; then \
+RUN if [ "${BUILD_DATE}" = "undefined" ]; then \
```
Comment (kkovaletp) [inline Dockerfile:32]: Isn't it better to fix it in the lines 19 and 20 by replacing the `undefined` with the `$(date -u +'%Y-%m-%dT%H:%M:%SZ')` and then remove this IF section?

Comment (googollee) [reply Dockerfile:32]: `ENV` command is not running in bash, but parses by Docker. It doesn't support bash scripts.

Comment (kkovaletp) [reply Dockerfile:32]: Ok, I see.

**Review by kkovaletp — APPROVED** [review]
Approved

**Pull request merged**